### PR TITLE
Updating multiple lessons

### PR DESCRIPTION
**Pull request by Benkex**
See my comments in the commits

### CODE REVIEW

File: content/tutorial/01-svelte/06-bindings/03-checkbox-inputs/README.md
Comment (dummdidumm) [inline content/tutorial/01-svelte/06-bindings/03-checkbox-inputs/README.md:1]: Please revert the changes to this exercise (and the one where you removed the shorthand explanation). The shorthand is explained at the end of that chapter - we don't need to move it in front - and we'd like to keep the `yes` name

File: content/tutorial/02-advanced-svelte/02-transitions/08-key-blocks/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: Key blocks
 ---
 
-Key blocks destroy and recreate their contents when the value of an expression changes. This is useful if you want an element to play its transition whenever a value changes instead of only when the element enters or leaves the DOM.
+Key blocks destroy and recreate their contents when the value of the following expression (in the example: `i`) changes. This is useful if you want an element to play its transition whenever a value changes instead of only when the element enters or leaves the DOM.
```
Comment (dummdidumm) [inline content/tutorial/02-advanced-svelte/02-transitions/08-key-blocks/README.md:5]: I find the previous sentence to be more readable, please revert this change. This is a general introductionary sentence to the key blocks.

File: content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ title: Media elements
 
 You can bind to properties of `<audio>` and `<video>` elements, making it easy to (for example) build custom player UI, like `AudioPlayer.svelte`.
 
-First, add the `<audio>` element along with its bindings (we'll use the shorthand form for `duration` and `paused`):
+First, add the `<audio>` element along with its bindings (we'll use the shorthand form for `src`, `duration` and `paused`, and to `src` we don't have to bind):
```
Comment (dummdidumm) [inline content/tutorial/02-advanced-svelte/05-bindings/03-media-elements/README.md:7]:
```suggestion
First, add the `<audio>` element along with its bindings (we'll use the shorthand form for `src`, `duration` and `paused`):
```

File: content/tutorial/02-advanced-svelte/05-bindings/06-component-bindings/app-a/src/lib/Keypad.svelte [hunk 3]
```diff
@@ -11,15 +11,9 @@
...
-	<button on:click={select(1)}>1</button>
-	<button on:click={select(2)}>2</button>
-	<button on:click={select(3)}>3</button>
-	<button on:click={select(4)}>4</button>
-	<button on:click={select(5)}>5</button>
-	<button on:click={select(6)}>6</button>
-	<button on:click={select(7)}>7</button>
-	<button on:click={select(8)}>8</button>
-	<button on:click={select(9)}>9</button>
+	{#each [1, 2, 3, 4, 5, 6, 7, 8, 9] as i}
```
Comment (dummdidumm) [inline content/tutorial/02-advanced-svelte/05-bindings/06-component-bindings/app-a/src/lib/Keypad.svelte:14]: While this is a clever way to write it, the previous version is more readable to novice Svelte devs because they don't have to understand `#each` blocks while solving the exercise - therefore please revert this change.

**Review by dummdidumm — CHANGES_REQUESTED** [review]
Thank you! I left a few comments with things I'd like to see reverted. In general it's easier most of the time to create separate PRs for separate exercises.

**Comment by Benkex** [comment]
No problem:) I'll do as you ask. I understand, why you want to revert the change with #each. But if that has to go, what about a small awareness-raising in the tutorial?
> In `Keypad.svelte` we've hidden some repetitive code, that can be reduced with Svelte syntax. Can you find and solve it?
Also, I find the revert of changing `yes` to `checked` sad, as yes is a "synonym" of true, which is a value, and as such, not ideal for the name of a variable in my opinion.
I will revert all requested changes tomorrow, and let you think again.
